cmds/core/yes/yes_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+func TestMain(m *testing.M) {
+	testutil.Run(m, main)
+}
+
+func TestYes(t *testing.T) {
+	cmd := testutil.Command(t)
+	fmt.Println(cmd.Path)
```

Comment:
don't use fmt.Print of any kind in a test, use t.Logf

Reply:
I realized the fmt.Println(cmd.Path) was not necessary to keep, so I just deleted that line of code :)!